tests/libres_tests/res/enkf/test_enkf_fs_manager2.py

```diff
@@ -28,6 +28,10 @@ def test_rotate(self):
             for index in range(EnkfFsManager.DEFAULT_CAPACITY):
                 fs_list.append(fsm.getFileSystem(f"fs_fill_{index}"))
 
+            # TODO: Why are the two first elements of `getCaseList` "default_0" and "default_1"?
```

Comment:
Could someone explain why this is so?

Reply:
If I understand the question correctly it is because there are two cases in that fs, look in: `ert/test-data/local/snake_oil/storage/snake_oil/ensemble`

Reply:
Also that the cases you have directly correspond to the directories in the `ENSPATH` (defaults to `storage/`). So the case `default_0` will have a directory named `default_0`, and vice-versa, if there exists a directory named `default_1` then ERT will assume that it contains the data for ensemble run named `default_1`.

Reply:
Thanks 👍